acq4/analysis/modules/STDPAnalyzer/STDPAnalyzer.py

```diff
@@ -0,0 +1,978 @@
+
+from PyQt4 import QtGui, QtCore
+from collections import OrderedDict
+from acq4.analysis.AnalysisModule import AnalysisModule
+import STDPControlTemplate, STDPPlotsTemplate
+import acq4.pyqtgraph as pg
+import numpy as np
+from acq4.util.functions import measureResistance
+from acq4.util.DatabaseGui.DatabaseGui import DatabaseGui
+import STDPFileLoader
+
+class STDPAnalyzer(AnalysisModule):
+
```

Comment:
Add a docstring here that describes the module. This will be displayed in the data manager.